### PR TITLE
Enforce IA - moving non-automated Enforce docs [PLEASE MERGE IF YOU WANT] 

### LINKED ISSUE #521: Enforce docs IA Overhaul
**Is your feature request related to a problem? Please describe.**
I'm always frustrated by how our Enforce docs are organized. It's very difficult to find what I'm looking for, and the left-hand sidebar on Academy is hard to read because the 

**Describe the solution you'd like**
I'm working on an overhaul of our Enforce docs IA, and creating this issue to track this work. I'm planning to shorten the titles of many tutorials to make them more legible, and hoping to make everything more discoverable.

absolute_nightmare

### PR DESCRIPTION
**Pull request by SharpRake**
## Type of change

This is my first solid try at moving around the enforce docs. ~This PR may get deleted quickly if the aliases don't work (they weren't working on my local testing).~ They work!

### What should this PR do?

resolves #521 and #534 

### Why are we making this change?

Let's make the docs look better!

### What are the acceptance criteria? 

I made some executive decisions regarding the organization. I'm very open to making updates as necessary.

### How should this PR be tested?

Confirm that aliases work and everything is laid out to your liking! 

EDIT: Be aware that the netlify warning about a redirect rule failing seems to be in error. Feel free to slack me if you have questions.

### CODE REVIEW

**Comment by SharpRake** [comment]
Netlify is behaving somewhat strangely - no matter what it says that one redirect rule can't be processed. Through independent testing I've confirmed that the `aliases` redirects in the Hugo front matter do work. If any don't work after merging I will fix them on an ad hoc basis.

File: content/chainguard/chainguard-enforce/changelog.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
 ---
 title : "Chainguard Enforce Changelog"
+linktitle: "Changelog"
+aliases:
+- /chainguard/chainguard-enforce/chainguard-enforce-kubernetes/changelog/
```
Comment (jamonation) [inline content/chainguard/chainguard-enforce/changelog.md:5]: This file comes from the Github workflow, so changes here will be lost. I think for this PR the easiest fix would be to leave the file in its original location, and we update the template that generates it, and the workflow separately?

File: content/chainguard/chainguard-enforce/chainguard-enforce-events/index.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ---
-title : "Chainguard Enforce Events"
+title: "Chainguard Enforce Events"
+linktitle: "Events"
+aliases:
+- /chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-events/
```
Comment (jamonation) [inline content/chainguard/chainguard-enforce/chainguard-enforce-events/index.md:5]: This file also comes from the Github workflow, so changes here will be lost. I think for this PR the easiest fix would be to leave the file in its original location, and we update the template that generates it, and the workflow separately?

**Review by jamonation — COMMENTED** [review]
Two notes about the changelog and events files. I'd personally leave them out for now, and we can update the workflow separately. Or, if you want we can add that change to this PR as well and do everything at once, up to you!

**Comment by SharpRake** [comment]
> Two notes about the changelog and events files. I'd personally leave them out for now, and we can update the workflow separately. Or, if you want we can add that change to this PR as well and do everything at once, up to you!

It seemed like updating the github worklfow wouldn't be too painful so I went that route. If something looks off let me know and I'll revert that change and we can figure it out in another PR.

**Comment by jamonation** [comment]
Looks good then, works for me! Since there aren't any aliases in the generated files, they'll need nginx redirects instead. Once those are done, +1!

**Comment by jamonation** [comment]
PR for nginx redirects against your branch is here @SharpRake: https://github.com/SharpRake/edu/pull/2

**Review by jamonation — APPROVED** [review]
Looks good to me 👍

**Pull request merged**